includes/stats/stats-importer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Wporg\TranslationEvents\Stats;
+
+use Wporg\TranslationEvents\Event\Event;
+
+class Stats_Importer {
+	/**
+	 * Imports the contributions a user made while a given event was active.
+	 */
+	public function import_for_user_and_event( int $user_id, Event $event ): void {
+		global $wpdb, $gp_table_prefix;
+
+		// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.DirectQuery
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.NoCaching
+		$wpdb->query(
+			$wpdb->prepare(
+				"
+				insert ignore into {$gp_table_prefix}event_actions
+				    (event_id, user_id, original_id, locale, action, happened_at)
+				select %d, t.user_id, t.original_id, ts.slug, %s, t.date_added
```

Comment:
```suggestion
				select %d, t.user_id, t.original_id, ts.locale, %s, t.date_added
```
This throws a notice `Notice: Trying to get property 'english_name' of non-object`

Reply:
Could you describe the steps you took so I can reproduce?

Reply:
These were the steps I took;

1. Log in as User A in browser A and create an event
2. Log in as User B in browser B and translate a string
3. Visit the event page for event created in #1 above and click 'Attend'.

The error shows in the stats table and projects section